### PR TITLE
Quick Fix on Docker based installation and support for AWS Fargate

### PR DESCRIPTION
**Pull request by rajeevr2715**
**Description**
Quick fix applied to fix the below issues,
1. https://github.com/tmobile/t-vault/issues/190
2. https://github.com/tmobile/t-vault/issues/191

Note: To run docker without --privileged option, Use, --cap-add=IPC_LOCK in docker run command and update the SKIP_SETCAP='true' in docker parameters

### CODE REVIEW

File: dist/src/main/components/web/nginx/conf/nginx.conf [hunk 1]
```diff
@@ -44,7 +44,7 @@ http {
 
         ssl_ciphers  HIGH:!aNULL:!MD5;
         ssl_prefer_server_ciphers  on;
-		
+
```
Comment (raghits) [inline dist/src/main/components/web/nginx/conf/nginx.conf]: Make sure we are not introducing unnecessary changes due to space/tabs or newlines across all files.

File: dist/src/main/docker/vault.sh [hunk 2]
```diff
@@ -1,28 +1,28 @@
 #! /bin/bash
 #set -x
-# =========================================================================		
+# =========================================================================
```
Comment (raghits) [inline dist/src/main/docker/vault.sh]: Several changes in this file as well owing to spaces and newlines. Please fix those.

File: dist/src/main/docker/vault.sh [hunk 3]
```diff
@@ -377,13 +378,16 @@ if [[ -z "$initstat" ]]; then
   echo "Initializing Vault..."
   echo "This only happens once when the server is started against a new backend that has never been used with Vault before."
   echo "During initialization, the encryption keys are generated and 5 unseal keys are created."
-
+  sudo setcap cap_ipc_lock=-ep $(readlink -f $(which vault))
```
Comment (raghits) [inline dist/src/main/docker/vault.sh]: Do we need to set this each time?

File: dist/src/main/components/web/nginx/conf/nginx.conf (hunk 1 above)
Comment (rajeevr2715) [reply dist/src/main/components/web/nginx/conf/nginx.conf]: Ok.reverted

File: dist/src/main/docker/vault.sh (hunk 2 above)
Comment (rajeevr2715) [reply dist/src/main/docker/vault.sh]: done

File: dist/src/main/docker/vault.sh (hunk 3 above)
Comment (rajeevr2715) [reply dist/src/main/docker/vault.sh]: yes. will get the error `/opt/tvault/hcorp/bin/vault: Operation not permitted` after each command.
Updated based on the parameter SKIP_SETCAP, To run docker without --privileged, use 'true' else 'false'
